src/india_api/internal/service/server.py

```diff
@@ -143,15 +196,15 @@ class GetForecastGenerationResponse(BaseModel):
     status_code=status.HTTP_200_OK,
 )
 def get_forecast_timeseries_route(
-    source: ValidSourceDependency,
-    region: str,
-    db: DBClientDependency,
+        source: ValidSourceDependency,
+        region: str,
+        db: DBClientDependency,
+        user: Auth0User = Security(get_user()),
+        # user: Auth0User = Security(get_user(), scopes=["read:india"]),
 ) -> GetForecastGenerationResponse:
     """Function for the forecast generation route."""
     values: list[PredictedPower] = []
 
-    db.save_api_call_to_db(url=f"/{source}/{region}/forecast", user=None)
```

Comment:
i'd probably keep this in for the moment, and use `get_user_by_email` - https://github.com/openclimatefix/pv-site-api/blob/main/pv_site_api/main.py#L297